### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by Eilon**
And also a few other Core-related renames.

@Tratcher @blowdart

### CODE REVIEW

File: src/Microsoft.Owin.Security.Interop/project.json [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-        "Microsoft.AspNetCore.DataProtection.Extensions": "1.0.0-*",
-        "Microsoft.Owin.Security": "3.0.1"
-    },
-    "frameworks": {
-        "net451": { }
-    }
+  "description": "A compatibility layer for sharing authentication tickets between Microsoft.Owin.Security and Microsoft.AspNetCore.Authentication.",
+  "tags": [
+    "aspnetcore",
+    "security"
```
Comment (PinpointTownes) [inline src/Microsoft.Owin.Security.Interop/project.json]: Is it worth adding `owin`? (and/or `katana`)

Comment (Eilon) [inline src/Microsoft.Owin.Security.Interop/project.json]: Yeah perhaps both. Seems like all the previous packages had those tags.

**Comment by Eilon** [comment]
:up: :date:

**Comment by Eilon** [comment]
https://github.com/aspnet/Security/commit/d78f7edfc7d3084c430eb129e3d6b8f703ff8eb9

**Comment by Eilon** [comment]
@PinpointTownes thanks!

**Pull request closed without merging**